bin/test-api.sh

```
@@ -0,0 +1,337 @@
+  local label="$1"
+  shift
+  local response http_code body
+
+  response=$(curl -s "${CURL_OPTS[@]}" -w "\n__HTTP_CODE__%{http_code}" \
+    -H "Authorization: Bearer $API_TOKEN" \
+    -H "Accept: application/json" \
+    -H "Content-Type: application/json" \
+    "$@")
+
+  http_code=$(echo "$response" | grep -o '__HTTP_CODE__[0-9]*' | sed 's/__HTTP_CODE__//')
+  body=$(echo "$response" | sed 's/__HTTP_CODE__[0-9]*$//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)